### PR TITLE
Add Certificate microservice functionality

### PR DESCRIPTION
**Pull request by angcabdom**

### CODE REVIEW

File: microservices/certificate/src/views.js [hunk 1]
```diff
@@ -1,31 +1,123 @@
...
+        delete: ['admin']
+      },
       methods: {
-        get: (req, res, next) => {
-          res.json({
-            test: 'mychild'
-          });
+        get: async (req, res) => {
+          try {
+            const record = await User.findById(req.params.certificateId);
```
Comment (0aps) [inline microservices/certificate/src/views.js]: here should be Certificate model

File: microservices/certificate/src/views.js [hunk 2]
```diff
@@ -1,31 +1,123 @@
...
+
+            if (!record) {
+              return next(new RecordNotFound());
+            }
+
+            const rs = fs.createReadStream('./' + data.certificatePath);
```
Comment (0aps) [inline microservices/certificate/src/views.js]: define data

File: microservices/certificate/src/views.js (hunk 1 above)
Comment (lucaspreel) [reply microservices/certificate/src/views.js]: Change done

File: microservices/certificate/src/views.js (hunk 2 above)
Comment (lucaspreel) [reply microservices/certificate/src/views.js]: Change done ("data" replaced by "record")

**Pull request merged**